### ISSUE TITLE
can't highlight codes after upgrading VS code to 1.54.2

### ISSUE THREAD

**Issue by Wing924** [jinliming2/vscode-go-template#5]
go-template plugin version: `0.2.0`

VS code info:
```
Version: 1.54.2
Commit: fd6f3bce6709b121a895d042d343d71f317d74e7
Date: 2021-03-11T00:56:09.783Z (4 days ago)
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin x64 19.6.0
```

**Comment by jinliming2** [comment]
Hi, I just published v0.2.1 to fix this, have a try.

**Comment by Wing924** [comment]
it works! thanks!
